packages/component-submission/server/resolvers/uploadSupportingFile.js

```diff
@@ -1,14 +1,15 @@
+const config = require('config')
 const User = require('@elifesciences/component-model-user').model
-const Manuscript = require('@elifesciences/component-model-manuscript').model
 const { S3Storage } = require('@elifesciences/component-service-s3')
-
-const { SupportingFiles } = require('../use-cases')
+const { SupportingFiles, Manuscript } = require('../use-cases')
 
 async function uploadSupportingFile(_, { file, id }, { user }) {
   const userUuid = await User.getUuidForProfile(user)
   const files = new SupportingFiles(S3Storage, id, userUuid)
   await files.upload(file)
-  const manuscript = await Manuscript.find(id, userUuid)
-  return manuscript
+
+  const manuscript = new Manuscript(config, userUuid, S3Storage)
+
+  return manuscript.getView(id)
```

Comment:
does this need an `await` ?

Reply:
async functions wrap result in a Promise anyway, so i don't think so....